UnityProject/Assets/Scripts/Systems/Lobby/CharacterManager.cs

```diff
@@ -0,0 +1,93 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Newtonsoft.Json;
+using UnityEngine;
+
+namespace Systems.Character
+{
+	/// <summary>Manage a player's characters. Intended for the local player client.</summary>
+	public class CharacterManager
+	{
+		public List<CharacterSheet> Characters { get; } = new();
+
+		private string OfflineStoragePath => $"{Application.persistentDataPath}characters.json";
+
+		public CharacterManager Init()
+		{
+			LoadOfflineCharacters();
+
+			return this;
+		}
+
+		public int GetCurrentCharacterIndex()
+		{
+			int lastCharacterIndex = PlayerPrefs.GetInt(PlayerPrefKeys.LastCharacterIndex);
+
+			return Math.Clamp(lastCharacterIndex, 0, Characters.Count);
+		}
+
+		/// <summary>Load characters that have been saved to the cloud.</summary>
+		public void LoadCharacters()
+		{
+			throw new NotImplementedException();
```

Comment:
not being used?

Reply:
No, the Firebase storage of characters hasn't been used since the character creator overhaul. It's a future task. As mentioned, this PR is just baby steps.